WordPress/Jetpack/Classes/Utility/DataMigrator.swift

```diff
@@ -62,23 +67,38 @@ final class DataMigrator {
         }
 
         copyTodayWidgetDataToJetpack()
+        copyShareExtensionDataToJetpack()
+        copyNotificationsExtensionDataToJetpack()
```

Comment:
I realized that the call site might not cover all the use cases (in fact, it'll miss the shot entirely). Copying the extensions' keychain and user defaults is not part of the WP->JP Migration but for fixing the data sharing issue. Here's why:

- In the case of WP->JP Migration, we only initiate the import when the JP app is in a logged-out state. 
- This means users already logged in to JP (and have potentially used the extensions) will not get their extensions' data migrated.
- However, since we already changed all the keys for the extensions' keychain and user defaults, logged-in users will lose access to their extensions.

However, I'm just noting this for myself as I'm the one who first put the `copyTodayWidgetDataToJetpack()` in the import method 😅. I will fix this later through a separate PR!